### DISCUSSION TITLE
Question: Usage of timezone field in CrontabSchedule

### DISCUSSION THREAD

**Discussion by laurent-brisbois** [celery/django-celery-beat#641] [Q&A] [answered]
### Summary:

Include a *brief* description of the problem here, and fill out the version info below.  

* Celery Version: `celery==5.2.7`
* Celery-Beat Version: `django-celery-beat==2.4.0`

### Exact steps to reproduce the issue:
By doing something like this, assuming the server is running in UTC :
```python
schedule, _ = CrontabSchedule.objects.get_or_create(
    minute="30",
    hour="10",
    day_of_month="*",
    month_of_year="*",
    day_of_week="1",
    timezone="Europe/Brussels",
)
```

### Detailed information

Is it correct to say that tasks with this schedule will run every Monday at 10:30 for the Europe/Brussels timezone ? (And on the server it'll run at 9:30 effectively since it is in UTC ?!)
Also does it take DST into account ?

Thanks in advance :slightly_smiling_face:

**Comment by devanshprophone** [comment, marked as answer]
Yes, it is correct to say that tasks with this schedule will run every Monday at 10:30 for the Europe/Brussels timezone, and on the server, it will run at 9:30 effectively since it is in UTC.

Regarding Daylight Saving Time (DST), it will be automatically taken into account by the pytz library, which is used by Celery to handle time zones. Europe/Brussels time zone observes DST, so when DST starts, the task will run at 11:30 UTC instead of 10:30 UTC, and when DST ends, the task will run at 9:30 UTC instead of 10:30 UTC.
